bundles/org.eclipse.jface/src/org/eclipse/jface/layout/AbstractColumnLayout.java

```diff
@@ -46,14 +46,14 @@
  * @since 3.4
  */
 public abstract class AbstractColumnLayout extends Layout {
-	private static int COLUMN_TRIM;
+	private static int columnTrim;
```

Comment:
Here, the goal with upper case is to emphasize that the value is in practice final and mustn't nor shouldn't be changed upon access. Please keep this convention.

Reply:
I have reverted the case change, however this scenario looks strange to me - The field cannot be declared static because it value neither known at compile time nor at construction time, and yet it holds a final value :)

Reply:
The value is known at class loading time, the `static { ... }` block after it will be executed as soon as class is loaded, and will set the value dynamically. It's not actually, but the name convention is a reminder that this static field must be consumed as if it were final.